shipyard_airflow/dags/drydock.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+### DryDock Dag
+"""
+import airflow
+from airflow import DAG
+from airflow.operators.sensors import HttpSensor
+from airflow.operators.bash_operator import BashOperator
+from datetime import timedelta
+import json
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': airflow.utils.dates.days_ago(2),
+    'retries': 1,
+    'retry_delay': timedelta(minutes=1),
+    'provide_context': True
+}
+
+dag = DAG('drydock_dag', default_args=default_args, schedule_interval="@once")
+
+# Location of site_yaml and prom_yaml
+site_yaml = '/home/ubuntu/demo-bundle/drydock.yaml'
+prom_yaml = '/home/ubuntu/demo-bundle/configs/complete-bundle.yaml'
+
+# Drydock Endpoint
+drydock_api_endpoint = 'http://drydock-api.drydock:9000'
+drydock_design_api = 'http://drydock-api.drydock:9000/api/v1.0/designs'
+drydock_task_api = 'http://drydock-api.drydock:9000/api/v1.0/tasks'
+
+# Drydock Token
+drydock_token = 'bigboss'
+
+
+# Add drydock connection to airflow
+add_connection = BashOperator(
```

Comment:
Based on what I seen on the internet, it seems that most people just add the new connection directly from the Web UI.

I agree that it is a bit odd to have airflow calling the CLI but I wasn't sure of other more elegant way of adding the drydock connection using automation.